examples/scripts/ejb-persistent-clustered-timers.cli

```diff
@@ -0,0 +1,4 @@
+batch
+/subsystem=ejb3/service=timer-service/database-data-store=ejb-timers-ds:add(datasource-jndi-name=${org.wildfly.datasources.postgresql.jndi-name,env.POSTGRESQL_JNDI:java:jboss/datasources/${org.wildfly.datasources.postgresql.datasource, env.POSTGRESQL_DATASOURCE,env.OPENSHIFT_POSTGRESQL_DATASOURCE:PostgreSQLDS}}, database=postgresql, partition=ejb-timers-ds-part)
```

Comment:
I am wandering if you could hardcode the datasource jndi name? The example relies on default values so should be: java:jboss/datasources/PostgreSQLDS